### PR TITLE
Build app

### PR DESCRIPTION
**Pull request by monshan**
# PR Rancid Tomatillos
**Connor Anderson-Larson & Marika Shanahan**

### Fix or Feature?

- [ ] Fix
- [x] Feature

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes

* Displays very simple props from mock data on UI
* Applies agreed styling and imports desired font
* Sets mock data as state following initial render
* Change App component from functional to class type
* Created the following components in a components folder
  * Theatre - Theatre.js, Theatre.css, functional comp
  * Poster - Poster.js, Poster.css, functional comp

### What ticket(s) do the changes resolve?

* Iteration 1 - User stories
  * All movies display on initial load

### Notes for the Reviewer

Please see comments that call attention to primary changes

### How to Test Changes?

open application in directory by running command `npm start`

### Next Iterations

* Considering Styling
  * Decide to incorporate Sassy CSS
* Keep in mind that the Poster component may need to be a class component due to the User Event eventually firing on those specific UI elements

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="A React web app that lets you browse movies from a local API"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!-- <meta name="author">Connor Anderson-Larson & Marika Shanahan</meta> -->
+    <meta name="title"></meta>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600;700&display=swap" rel="stylesheet">
```
Comment (monshan) [inline public/index.html:15]: import font

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor () {
+    super();
+    this.state = {
+      movies: movieData.movies
+    }
```
Comment (monshan) [inline src/App.js:11]: sets initial state as mock data

File: src/components/Poster/Poster.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// import { Component } from "react";
+
+// class Poster extends Component {
+//   constructor() {
+//     super();
+//     this.state = {
+//       ""
+//     }
+//   }
+// }
```
Comment (monshan) [inline src/components/Poster/Poster.js:10]: left commented for future use until deemed necessary

File: src/components/Theatre/Theatre.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.theatre {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
```
Comment (monshan) [inline src/components/Theatre/Theatre.css:3]: fluent use of css grid

File: src/App.css [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+main {
+  background-color: #64284B;
+}
+
+.App {
```
Comment (ConnorAndersonLarson) [inline src/App.css:9]: Change class name in future to match naming conventions

File: src/App.js [hunk 6]
```diff
@@ -1,25 +1,31 @@
...
+  }
+
+  render () {
+    return (
+      <div className="App">
```
Comment (ConnorAndersonLarson) [inline src/App.js:16]: change className to match naming convention

File: src/components/Poster/Poster.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+import './Poster.css'
+
+const Poster = ({ id, poster_path, backdrop_path, title, average_rating, release_date }) => {
+  return (
+    <section id={id} className="poster">
+      <img src={poster_path} alt="Movie poster" className="movie-poster" />
+      <div className="movie-info">
+        <h2>{ title }</h2>
+        <p>{ release_date }</p>
```
Comment (ConnorAndersonLarson) [inline src/components/Poster/Poster.js:20]: Maybe add a 'Released:' { release_date } for clarity later on

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks good so far! Definitely some non-essential styling things we can look into later (BEM, normalize/reset, etc.) but for this PR it's golden.

**Pull request merged**